ArduPlane/Plane.h

```diff
@@ -261,6 +264,9 @@ class Plane : public AP_HAL::HAL::Callbacks {
     // selected navigation controller
     AP_SpdHgtControl *SpdHgt_Controller = &TECS_controller;
 
+    //Soaring Controller
+    SoaringController soaring_controller {ahrs, SpdHgt_Controller, aparm, &DataFlash, LOG_THERMAL_MSG, LOG_VARIO_MSG};
```

Comment:
You may have look at the wrong PR, I fixed the DataFlash issue in #5737. As for log messages macros, I don't see any examples of other libraries trying to use the messages defined in ArduPlane/APMRover2/ArduCopter